### PR TITLE
truly working responsive side nav

### PR DESCRIPTION
**Pull request by MichelML**
this version works flawlessly in the admin, demo incoming  

sidenav2

### CODE REVIEW

File: scss/components/header.scss [hunk 1]
```diff
@@ -6,20 +6,41 @@
...
   }
 }
 
+
 .header-height {
   height: $panel-header-height;
 }
+
+.header-hamburger {
+  padding: 15px 20px;
```
Comment (sroy3) [inline scss/components/header.scss]: Don't forget to move this to variables

File: scss/components/header.scss [hunk 2]
```diff
@@ -6,20 +6,41 @@
...
+
+  background-color: transparent;
+  border: none;
+
+  outline-width: 0;
+  cursor: pointer;
+
+  transition: opacity $navigation-toggle-duration ease-in-out;
+
+  opacity: 0.7;
```
Comment (sroy3) [inline scss/components/header.scss]: I think there are variables for opacity. It's `$transparency-1(...)` or something like that.

Comment (GermainBergeron) [inline scss/components/header.scss]: There is some variable for transparencies, see variables.scss

File: scss/components/header.scss (hunk 1 above)
Comment (GermainBergeron) [inline scss/components/header.scss]: I hate to be this guy, but it should be variables

File: scss/variables.scss [hunk 3]
```diff
@@ -226,6 +226,14 @@ $navigation-loading-height: 12px;
 $navigation-loading-item-border-radius: 10px;
 $navigation-loading-background-size: 300% 104px;
 $navigation-loading-bullet-width: 12px;
+$navigation-toggle-duration: 0.3s;
+%navigation-toggle-closed {
+  margin-left: -$navigation-width;
+  transition: transform $navigation-toggle-duration ease-in-out; 
+}
+%navigation-toggle-opened {
```
Comment (GermainBergeron) [inline scss/variables.scss]: Those (%navigation-toggle-closed and %navigation-toggle-opened) are used in v2?

Comment (MichelML) [reply scss/variables.scss]: yes for status and quick nav which are outside the nav bar

**Pull request merged**